openmls/src/group/mod.rs

```diff
@@ -41,11 +40,7 @@ pub use public_group::*;
 pub(crate) use core_group::create_commit_params::*;
 #[cfg(any(feature = "test-utils", test))]
 pub(crate) mod tests;
-#[cfg(any(feature = "test-utils", test))]
```

Comment:
Why remove these features?

Reply:
Wait. Now that I'm running the repo locally I see the same warnings. Going to merge this so that I can pick up the fixes in my branch.